pyproject.toml

```diff
@@ -42,6 +42,8 @@ sphinx-rtd-theme = {version = "^1.0.0", optional = true}
 readthedocs-sphinx-ext = {version = "^2.1.8", optional = true}
 m2r2 = {version = "^0.3.2", optional = true}
 sphinx-autoapi = {version = "^1.8.4", optional = true}
+requests-toolbelt = "^0.9.1"
+cryptography = "^37.0.4"
```

Comment:
This is only used in unit tests, so should be part of the dev-dependencies

Reply:
Ok I am correcting cryptography moving it to dev dependencies. Instead requests-toolbelt is used also in a function that I have added.